### DISCUSSION TITLE
[Feature Request] Reaquest to add timeline

### DISCUSSION THREAD

**Discussion by kdipto1** [heroui-inc/heroui#1906] [Feature requests]
### Is your feature request related to a problem? Please describe.

The timeline displays a list of events in chronological order.

### Describe the solution you'd like

The timeline displays a list of events in chronological order.

### Describe alternatives you've considered

The timeline displays a list of events in chronological order.

### Screenshots or Videos

![image](https://github.com/nextui-org/nextui/assets/96812332/e22649ae-5dca-4104-9ef9-5bfcfad2aeb5)

**Comment by MuaazOsaidTahir** [comment]
Can we get some ETA for this feature?
